bin/internal/engine.version

```diff
@@ -1 +1 @@
-53c27e519d33b4e13b01a8710b38a3591d6ca6f1
+b677f7600a43f6ae5c211b7d8d1675258dcbf591
```

Comment:
You want to make this in a follow up change. The revision b677f7600a43f6ae5c211b7d8d1675258dcbf591 will never be built in post-submit (PRs get squashed before merged, and engine binaries don't get built until post-submit).